frontend/www/js/omegaup/linkable_resource.ts

```diff
@@ -161,3 +161,55 @@ export class SchoolUser implements LinkableResource {
     return Optional.ofNonNull(`${this.getDisplayValue()}`);
   }
 }
+
+export class Contest implements LinkableResource {
+  alias: string = '';
+  title: string = '';
+
+  constructor(contest: types.Contest) {
+    this.alias = contest.alias;
+    this.title = contest.title;
+  }
+
+  toString(): string {
+    return this.title;
+  }
+
+  getUrl(): string {
+    return `/arena/${this.alias}/`;
+  }
+
+  getBadge(): Optional<string> {
+    return Optional.empty();
+  }
+
+  getLogo(): Logo | null {
+    return null;
+  }
+}
+
+export class Course implements LinkableResource {
+  alias: string = '';
+  name: string = '';
```

Comment:
```suggestion
  alias: string;
  name: string;
```

aquí no es necesario agregar los defaults porque se están estableciendo siempre en el constructor. igual arriba.